maintenance/wikia/migrateImagesBetweenSwiftDC.php

```diff
@@ -12,17 +12,17 @@
  * Maintenance script class
  */
 class MigrateImagesBetweenSwiftDC extends Maintenance {
-	
+
 	/* @private Int - number of images to sync */
 	private $mLimit;
 	/* @private String - destination DC */
 	private $mDC_dst;
 	/* @private Array - containers */
-	private $source_container;
-	private $dest_container;
-	/* @private \Queue object  */
+	private $source_container = [];
+	private $dest_container = [];
```

Comment:
True, true - however, I don't want to rewrite the whole script :) These are just minor improvements of errors handling and logging.